src/exceptions.py

```diff
@@ -1,16 +1,30 @@
+class InvalidJSON(Exception):
```

Comment:
Adding a wider range of exception classes so we can handle them in the responses